plugins/sendBotMessage.js

```diff
@@ -41,6 +41,7 @@ export default async function sendBotMessage(
     },
     body: JSON.stringify({
       robot_jid: options.botJid,
+      user_jid: userJid,
```

Comment:
what's this change?

Reply:
The requests to send messages were failing with a "Invalid request body format" message. I've found a few references in their dev forum pointing to this field that should be added, probably a new field that has become required I believe: https://marketplace.zoom.us/docs/guides/team-chat-apps/send-edit-and-delete-messages/#send-messages